aws/logs_monitoring/settings.py

```diff
@@ -207,7 +207,14 @@ def __init__(self, name, pattern, placeholder):
         CiphertextBlob=base64.b64decode(ENCRYPTED)
     )["Plaintext"]
     if type(DD_API_KEY) is bytes:
-        DD_API_KEY = DD_API_KEY.decode("utf-8")
+        # If the CiphertextBlob was encrypted with AWS CLI, we
+        # need to re-encode this in base64
+        try:
+            DD_API_KEY = DD_API_KEY.decode("utf-8")
+        except:
```

Comment:
You'd get something like `[ERROR] UnicodeDecodeError: 'utf-8' codec can't decode byte 0x86 in position 1: invalid start byte 1`  which isn't super meaningful

Reply:
Ok, I just meant to avoid trying this encode/decode operation if the error was something else. What is the exception type we catch when we're unable to decode from utf-8 directly?

Reply:
You're right, I'm updating the PR